### ISSUE TITLE
on v.0.55 {{ .Content }} can behave wrongly

### ISSUE THREAD

**Issue by divinerites** [gohugoio/hugo#5827]
Tested on hugo v.0.55-DEV

- Site with Uno theme : https://themes.gohugo.io/hugo-uno/
- On v0.55-DEV everything compile fine and few WARNING/deprecated I removed. So it was green light for me.
- I just tested it in server mode and i have a weird addition of `<pre><code>` plus what looks like encoding process.
- None of my code or theme code has any test or whatever with ` <pre><code>`

I guess it seems related to `{{  .Content }}` behaviour.

#### Good HTML with v0.54
```HTML
<h1>Contrats / code Branche</h1>
<section id="texteintro">
--
  |  
  |  
  | <p class="info" align="center">... [MODE SERVEUR HUGO] ...<p>
  | <p class="info">Fichier à traiter : csv/Portefeuille_02.csv</p>
  | <p class="info">N° de colonne à aggréger : 9</p>
  | <p class="info">Type de graphique par défaut : line</p>
  | </section>
  |  
```

#### bad HTML vwith v0.55-DEV
```HTML
<h1>Contrats / code Branche</h1>
<pre><code>    &lt;section id=&quot;texteintro&quot;&gt;
--
  |  
  |  
  | &lt;p class=&quot;info&quot; align=&quot;center&quot;&gt;... [MODE SERVEUR HUGO] ...&lt;p&gt;
  | &lt;p class=&quot;info&quot;&gt;Fichier à traiter : csv/Portefeuille_02.csv&lt;/p&gt;
  | &lt;p class=&quot;info&quot;&gt;N° de colonne à aggréger : 9&lt;/p&gt;
  | &lt;p class=&quot;info&quot;&gt;Type de graphique par défaut : line&lt;/p&gt;
  | &lt;/section&gt;
```

This is the part where this code is generated.

```go-html-template
    {{ partial "head.html" . }}
    <body>
        {{ partial "sidebar.html" . }}
        <div class="content-wrapper">
            <div class="content-wrapper__inner">
                <div class="post">
                    <h1>{{ .Title }}</h1>
#### PROBLEM IS FROM THERE
                    {{ .Content }}
                </div>
            </div>
        </div>
    </body>
    {{ partial "script.html" .}}
</html>
```

**Comment by bep** [comment]
This is mostly related to the `%}}` shortcode delimiter change -- which is deliberate. But you need to post a site that demonstrates this problem for me to do more than guessing.

**Comment by divinerites** [comment]
Yes I see where. Makes sense.
Will post when I have access to my computer.

**Comment by bep** [comment]
I will close this. Please open another issue if this is still relevant.

**Comment by divinerites** [comment]
Adding {{ $_hugo_config := `{ "version": 1 }` }} didn't solved it.

**Comment by bep** [comment]
@divinerites I didn't receive any zip. That said, I'm not a big fan of doing open source development via zip files in my email in box -- so I will let this issue linger. If it's a real problem, then there will eventually be a person that can point to a public repo.

**Comment by divinerites** [comment]
OK. Sorry about that.

I will create a repo without the personal data. Will update the issue then.

Regards.

—
Didier G.
divinerites@gmail.com

Le 12 avril 2019 à 09:38:59, Bjørn Erik Pedersen (notifications@github.com)
a écrit:

@divinerites <https://github.com/divinerites> I didn't receive any zip.
That said, I'm not a big fan of doing open source development via zip files
in my email in box -- so I will let this issue linger. If it's a real
problem, then there will eventually be a person that can point to a public
repo.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/gohugoio/hugo/issues/5827#issuecomment-482472166>, or mute
the thread

**Comment by divinerites** [comment]
Again Bjorn sorry for not doing this before.

This is a minimal repo showing the problem in 0.55 and working in 0.54


https://github.com/divinerites/mytest

—
Didier G.
divinerites@gmail.com

Le 12 avril 2019 à 09:52:18, Didier Divinerites (divinerites@gmail.com) a
écrit:

OK. Sorry about that.

I will create a repo without the personal data. Will update the issue then.

Regards.

—
Didier G.
divinerites@gmail.com

Le 12 avril 2019 à 09:38:59, Bjørn Erik Pedersen (notifications@github.com)
a écrit:

@divinerites <https://github.com/divinerites> I didn't receive any zip.
That said, I'm not a big fan of doing open source development via zip files
in my email in box -- so I will let this issue linger. If it's a real
problem, then there will eventually be a person that can point to a public
repo.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/gohugoio/hugo/issues/5827#issuecomment-482472166>, or mute
the thread

**Comment by divinerites** [comment]
Solved in 0.55.4 with https://github.com/gohugoio/hugo/pull/5899
